### DISCUSSION TITLE
Suggestions for a 128x64 display that fits inside a GoTek Artery case

### DISCUSSION THREAD

**Discussion by RetroJeff** [keirf/flashfloppy#628] [Q&A]
Looking for suggests of a 128x64 display that will fit inside a GoTek Artery case ?
I have 3 different ones and either they don't work (90 degree rotated) or don't fit in the case.
-- Jeff

**Comment by keirf** [comment]
Get a factory model with oled fitted. Or carefully unstick the panel from its PCB and lay the PCB horizontally (works because panel is connected via flexible ribbon)

**Comment by mpaeke** [comment]
What about an enhancement of the -rotate option to rotate display by 90 degrees? There are OLED displays with 64x128 resolution available that fits perfect in the gotek case. Unfortunately they need a display rotation of 90° so they can't be used with flashfloppy.

Display type:
GME64128-II
0.96" VER64x128 VER:3.2
SSD1107

Product pictures suggest a horizontal use (128×64) but it's vertical (64×128) without rotation.
